src/AsyncGenerator.yml

```diff
@@ -96,6 +94,9 @@
     - conversion: Ignore
       name: Exists
       containingTypeName: AbstractCollectionPersister
+    - conversion: Ignore
+      name: QuoteTableAndColumns
+      containingTypeName: SchemaMetadataUpdater
```

Comment:
This method was obsoleted in 5.0, and did not have an async counterpart, so ignore it explicitly.